docs/UserGuide.md

```diff
@@ -38,7 +38,26 @@ _text_ | Caption for images.
 :bulb: | Indication that the following text is a tip.
 :exclamation: | Indication that the following text is important.
 
+### Command Format
+Commands are text that you can enter into TAB's command box to tell TAB which operation you would like to perform. Commands in TAB have several components and follow a general format as follows:<br/>
 
+```COMMAND_WORD PREAMBLE PARAMETER_PREFIX/PARAMETER…​```
+
+Detailed notes about the command format can be found [here](#features).
+
+|Component|Description|
+|---------|-----------|
+|Command word|The word that specifies which action you want TAB to execute.|
+|Preamble| The text before the first valid parameter prefix in the command. The preamble is where you would specify the student or lesson indices for commands that require them. The preamble may or may not be empty depending on the command.|
+|Parameter prefix| The specific prefix that identifies the start of the parameter. Each parameter has a unique prefix, but all prefixes end with a `/`.|
+|Parameter| An input provided by the user for the execution of the command.|
```

Comment:
Add in `Parameter value` can? replace instances of the word `argument` with `value` or `parameter value`

Reply:
In the Notes about the command format section it goes PREFIX/PARAMETER so the parameter is the value?

Reply:
oh icic ok